into/backends/sql_csv.py

```diff
@@ -83,6 +83,38 @@ def copy_mysql(dialect, tbl, csv):
     return statement.format(**locals())
 
 
+try:
+    import boto
+    from into.backends.aws import S3
+    from redshift_sqlalchemy.dialect import CopyCommand
+except ImportError:
+    pass
+else:
+    # TODO: kwargs passing? there are some dialect specific things that might
+    # be useful here
+    @copy_command.register('redshift.*')
+    def copy_redshift(dialect, tbl, csv, schema_name='public'):
```

Comment:
Can the schema_name be grabbed somehow from the `tbl`?  Perhaps from the associated engine (`tbl.bind`)?